source/ranges.tex

```diff
@@ -33,154 +33,154 @@
+    inline constexpr bool disable_sized_range = false;                              // freestanding
 
   template<class T>
-    concept sized_range = @\seebelow@;
+    concept sized_range = @\seebelow@;                                                // freestanding
 
   // \ref{range.view}, views
   template<class T>
-    inline constexpr bool enable_view = @\seebelow@;
+    inline constexpr bool enable_view = @\seebelow@;                                  // freestanding
 
-  struct view_base {};
+  struct view_base {};                                                              // freestanding
 
   template<class T>
-    concept view = @\seebelow@;
+    concept view = @\seebelow@;                                                       // freestanding
 
   // \ref{range.refinements}, other range refinements
   template<class R, class T>
-    concept output_range = @\seebelow@;
+    concept output_range = @\seebelow@;                                               // freestanding
 
   template<class T>
-    concept input_range = @\seebelow@;
+    concept input_range = @\seebelow@;                                                // freestanding
 
   template<class T>
-    concept forward_range = @\seebelow@;
+    concept forward_range = @\seebelow@;                                              // freestanding
 
   template<class T>
-    concept bidirectional_range = @\seebelow@;
+    concept bidirectional_range = @\seebelow@;                                        // freestanding
 
   template<class T>
-    concept random_access_range = @\seebelow@;
+    concept random_access_range = @\seebelow@;                                        // freestanding
 
   template<class T>
-    concept contiguous_range = @\seebelow@;
+    concept contiguous_range = @\seebelow@;                                           // freestanding
 
   template<class T>
-    concept common_range = @\seebelow@;
+    concept common_range = @\seebelow@;                                               // freestanding
 
   template<class T>
-    concept viewable_range = @\seebelow@;
+    concept viewable_range = @\seebelow@;                                             // freestanding
 
   // \ref{view.interface}, class template \tcode{view_interface}
   template<class D>
     requires is_class_v<D> && @\libconcept{same_as}@<D, remove_cv_t<D>>
-  class view_interface;
+  class view_interface;                                                             // freestanding
 
   // \ref{range.subrange}, sub-ranges
-  enum class subrange_kind : bool { unsized, sized };
+  enum class subrange_kind : bool { unsized, sized };                               // freestanding
 
   template<@\libconcept{input_or_output_iterator}@ I, @\libconcept{sentinel_for}@<I> S = I, subrange_kind K = @\seebelow@>
     requires (K == subrange_kind::sized || !@\libconcept{sized_sentinel_for}@<S, I>)
-  class subrange;
+  class subrange;                                                                   // freestanding
```

Comment:
Why is this diff missing the `ranges::get` overloads for `subrange` added by f400d80927fd580f99f5f2d94c3d07eaa47373d0?

Reply:
Does this require any action now?

Reply:
Yes, #5838